core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -113,6 +114,10 @@ oppia.directive('audioTranslationBar', [
           $scope.checkingMicrophonePermission = false;
           $scope.audioTimerIsShown = true;
           $scope.audioIsCurrentlyBeingSaved = false;
+          $scope.elapsedTime = 0;
+          $scope.timerInterval = null;
+          $scope.isPlayingUnsaved = false;
```

Comment:
We try to only start with is/are for function names since in general function names start with verbs to help distinguish them from variables. Maybe we can rename this to `unsavedAudioIsPlaying` or something around this?

Reply:
ok will do that.